sdk_container/src/third_party/portage-stable/sys-boot/grub/grub-2.12-r4.ebuild

```diff
@@ -88,9 +77,6 @@ IUSE="device-mapper doc efiemu +fonts mount nls sdl test +themes truetype libzfs
 
 GRUB_ALL_PLATFORMS=( coreboot efi-32 efi-64 emu ieee1275 loongson multiboot
 	qemu qemu-mips pc uboot xen xen-32 xen-pvh )
-
-# Flatcar: Add arm64 to the list of platforms
-GRUB_ALL_PLATFORMS+=( arm64 )
```

Comment:
Hm, are there no problems with missing `arm64` platform? Or it isn't really necessary now?